Buzz_gazebo/include/BuzzBridge.hpp

```diff
@@ -0,0 +1,34 @@
+#ifndef BUZZBRIDGE_HPP
```

Comment:
Do we have a reason to use .hpp instead of .h? Maybe this will be discussed in our formating meeting?